treeherder/log_parser/parsers.py

```diff
@@ -359,35 +359,25 @@ class ErrorParser(ParserBase):
         "FATAL ERROR",
         "REFTEST ERROR",
         "PROCESS-CRASH",
-        "Assertion failure:",
-        "Assertion failed:",
+        "Assertion fail",
         "###!!! ABORT:",
-        "E/GeckoLinker",
```

Comment:
This might still be found in some logcats, but I think it should not be highlighted by treeherder -- glad to see it removed.